### PR TITLE
Bugfix/sentry performance monitoring

### PR DESCRIPTION
**Pull request by sandisunandar99**
## Overview 

- installing monitoring performance sentry for hapi 16

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -14,23 +14,35 @@ const formatResponse = (response) => {
...
+    const transaction = Sentry.startTransaction({
+      op : "transaction",
+      name : response.request.path
+    })
+
+    Sentry.configureScope(scope => {
+      scope.setSpan(transaction)
+    })
+
+
```
Comment (firmanJS) [inline api/index.js:29]: not middleware using transactions for hapi ?? nice dude

File: config/config.js [hunk 2]
```diff
@@ -30,9 +33,12 @@ conf.sentry = {
...
   integrations: [
     new Sentry.Integrations.Http({ tracing: true }),
-    new Tracing.Integrations.Mongo
+    new Tracing.Integrations.Mongo(),
+    new Tracing.Integrations.Express({
+      server
+    })
   ],
-  tracesSampleRate: process.env.SENTRY_TRACE_RATE,
+  tracesSampleRate: 1.0
```
Comment (firmanJS) [inline config/config.js:41]: recomended using env file for dynamic code bro example `Number(process.env.SENTRY_TRACE_RATE)`

**Review by firmanJS — CHANGES_REQUESTED** [review]

**Review by khihadysucahyo — APPROVED** [review]

File: config/config.js (hunk 2 above)
Comment (sandisunandar99) [reply config/config.js]: oke done

**Review by firmanJS — APPROVED** [review]

File: api/index.js (hunk 1 above)
Comment (sandisunandar99) [reply api/index.js:29]: in index.js file inside folder api, there is fungction onRequest and onResponse that is function for traping all request and response. and I think, I call this function is middleware for old hapijs version 😆

**Pull request merged**